common/tasks/main.yml

```diff
@@ -34,46 +34,39 @@
 - name: Install common binary tools
   homebrew_cask: name={{item}}
   with_items:
-    - hipchat
-    - adium
-    - teamviewer
-    - skype
     - google-hangouts
     - firefox
-    - thunderbird
     - google-chrome
     - iterm2
     - sublime-text
     - macdown
     - atom
-    - dropbox
-    - google-drive
     - dash
     - disk-inventory-x
     - keepassx
-    - mattermost
```

Comment:
this is still used in day-to-day work!

Reply:
But on Natalia's Mac it was pre installed by IT - which lets the script crash because the app is not installed via homebrew

Reply:
I suppose it is now a standard app so we can assume it is always there already

Reply:
In anyway I suppose these packages are not mandatory and can be install by demand if needed.

Reply:
Let's try to extract mattermost into it's own task with `ignore_errors: yes`, see http://docs.ansible.com/ansible/latest/playbooks_error_handling.html